website/sidebars.js

```diff
@@ -22,7 +22,7 @@ const sidebars = {
 			className: 'sidebarHeader',
 			collapsed: false,
 			collapsible: false,
-			items: ['start', 'file_based', 'yubihsm', 'azure_kms', 'gcp_kms', 'aws_kms', 'ledger', `cli`, 'remote_policy', 'bakers'],
+			items: ['start', 'file_based', 'yubihsm', 'azure_kms', 'gcp_kms', 'aws_kms', 'ledger', `cli`, 'remote_policy', 'architecture', 'bakers'],
```

Comment:
Do we need this doc for all users? This document link can be given in getting started, and mention that it is optional for anyone who wants a more profound understanding of signatory.

Reply:
yes we need for all users who include reviewers